viadot/task_utils.py

```diff
@@ -106,6 +108,42 @@ def update_dtypes_dict(dtypes_dict):
     return dtypes_dict_updated
 
 
+@task
+def df_to_csv(
+    df: pd.DataFrame, path: str, sep="\t", if_exists="replace", **kwargs
+) -> None:
+    if if_exists == "append":
+        if os.path.isfile(path):
+            csv_df = pd.read_csv(path)
+            out_df = pd.concat([csv_df, df])
+        else:
+            out_df = df
+    elif if_exists == "replace":
+        out_df = df
+    out_df.to_csv(path, index=False, sep=sep)
+
+
+@task
+def df_to_parquet(
+    df: pd.DataFrame, path: str, if_exists: str = "replace", **kwargs
+) -> None:
+    if if_exists == "append":
+        if os.path.isfile(path):
+            parquet_df = pd.read_parquet(path)
+            out_df = pd.concat([parquet_df, df])
+        else:
+            out_df = df
+    elif if_exists == "replace":
+        out_df = df
```

Comment:
The way it's written it can be simplified:
```python
if if_exists == "append" and os.path.isfile(path):
    parquet_df = pd.read_parquet(path)
    out_df = pd.concat([parquet_df, df])
else:
    out_df = df
```
as long as we don't need to handle any other possible value. However, AFAIK we also have a possible value of `skip`? This is why `if_exists` should by typed as `Literal["append", "replace", "skip", etc...]` so that it's easy to see what are the valid options - and probably all of them should be handled, eg. skipping should probably produce a log message.